src/mesh/MeshService.cpp

```diff
@@ -85,8 +85,9 @@ int MeshService::handleFromRadio(const meshtastic_MeshPacket *mp)
                                                                               //  ignore our request for its NodeInfo
     } else if (mp->which_payload_variant == meshtastic_MeshPacket_decoded_tag && !nodeDB->getMeshNode(mp->from)->has_user &&
                nodeInfoModule) {
-        LOG_INFO("Heard new node on channel %d, sending NodeInfo and asking for a response.", mp->channel);
-        if (airTime->isTxAllowedChannelUtil(true)) {
+        if (airTime->isTxAllowedChannelUtil(true) && !IS_ONE_OF(config.device.role, meshtastic_Config_DeviceConfig_Role_ROUTER,
+                                                                meshtastic_Config_DeviceConfig_Role_REPEATER)) {
+            LOG_INFO("Heard new node on channel %d, sending NodeInfo and asking for a response.", mp->channel);
             nodeInfoModule->sendOurNodeInfo(mp->from, true, mp->channel);
         } else {
```

Comment:
This will create a lot of logs on routers saying "Skip sending NodeInfo due to > 25 percent channel util."

Change to "Skip sending NodeInfo due to > 25 percent channel util or we are are ROUTER"? ;)

Reply:
Good catch!